pulp_rpm/app/models/package.py

```diff
@@ -34,31 +32,6 @@ def db_type(self, connection):
         return "pulp_evr_t"
 
 
-class PackageManager(ContentManager):
```

Comment:
The rationale for getting rid of this is that different implementations of `with_age()` are not stable when applying additional filters to the queryset.  You want the annotation to be the very last thing you do post-filtering to guarantee correctness.